scripts/yast2

```diff
@@ -285,20 +284,16 @@ else
 	shift
 fi
 
-# Pass the arguments to WFM::Args():
-# Surround all args by (...).  args may contain spaces.
-# Use -S to avoid YCP escaping (bnc#382883)
-# But avoid a lone -S (cc-gnome dies, bnc#537470)
+# Pass the arguments to client:
 
 if [ -n "$CMDLINE_HELP" ]; then
     set -- $@ help;
 fi
-if [ $# -gt 0 ]; then
-    a=("$@")
-    b=("${a[@]/#/(}")
-    c=("${b[@]/%/)}")
-    set -- -S "${c[@]}"
-fi
+ARGS=""
+for i in "$@"; do
+  ARGS="$ARGS --arg `printf %q "$i"`"
```

Comment:
Well, my earlier suggestion, and this variant, are both wrong.
We need tests for this.